### PR TITLE
Readme, setup: Modify

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Readme is now changed to .rst and package has a new
description.

@sils1297 take a look and merge it if you think everything's okay, and ill make the package afterwards

### CODE REVIEW

File: coala-decorators/Decorators.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
+import inspect
```
Comment (sils) [inline coala-decorators/Decorators.py:1]: why are you adding this file a second time?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+# coala-decorators
```
Comment (sils) [inline README.rst:1]: we already have a readme.md

File: setup.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
   name = 'coala-decorators',
   packages = ['coala-decorators'],
   version = '0.1',
-  description = 'All the coala decorators',
+  description = 'A collection of useful decorators.',
```
Comment (sils) [inline setup.py]: own commit

File: README.rst (hunk 2 above)
Comment (Adrianzatreanu) [inline README.rst:1]: well i wanted to make it rst as this is how we use it everywhere?

File: coala-decorators/Decorators.py (hunk 1 above)
Comment (Adrianzatreanu) [inline coala-decorators/Decorators.py:1]: i moved it to a different folder, to package everything. i dont know why but it seems that i need to have the files that i want packaged in a different folder that is placed where the readme, license, setups are.

**Comment by sils** [comment]
@rultor merge

**Pull request merged**